views/partials/head.ejs

```diff
@@ -36,7 +36,7 @@
 	else if (language == 'es') ogp_locale = 'es_ES'
 	else if(language == 'pt') ogp_locale = 'pt_PT'
 
-	const nonce = locals.metadata?.page?.nonce
+	const nonce = locals?.nonce
```

Comment:
can this please be passed through locals.metadata.site.nonce (in datastructures.js), instead of simply locals.nonce?